style.css

```diff
@@ -206,3 +208,334 @@ li {
+.card-Work-container {
+  width: 100%;
+  height: 580px;
+  background-color: #ffff;
+  border: 2px solid #dfe1e6;
+  border-radius: 16px;
+  margin-top: 100px;
+}
+
+.card-Work-container-2 {
+  width: 100%;
+  height: 630px;
+  background-color: #ffff;
+  border: 2px solid #dfe1e6;
+  border-radius: 16px;
+  margin-top: 100px;
+}
+
+.snapshot-porfolio img {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 100px;
+}
+
+.html {
+  width: 53px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+.cascade {
+  width: 45px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+.javascript {
+  width: 89px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+.tag {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  color: #6070ff;
+  display: flex;
+  gap: 10px;
+  margin-left: 60px;
+  margin-top: 20px;
+}
+
+.card-Work-container h2 {
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  margin-left: 58px;
+  margin-top: 10px;
+}
+
+.card-Work-container-2 h2 {
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  margin-left: 58px;
+  margin-top: 10px;
+}
+
+.canopy {
+  margin-left: 5px;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  margin-top: 5px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.tonic {
+  display: flex;
+  gap: 20px;
+  margin-left: 58px;
+}
+
+.back-end {
+  list-style-image: url(./TAG/Counter.png);
+  color: #7a869a;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  margin-top: 5px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.year {
+  list-style-image: url(./TAG/Counter.png);
+  color: #7a869a;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  margin-top: 5px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.card-Work-container p {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin-left: 58px;
+  margin-top: 10px;
+}
+
+.card-Work-container-2 p {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin-left: 58px;
+  margin-top: 10px;
+}
+
+.see-project-one {
+  margin-top: 30px;
+  margin-right: 120px;
+  width: 400px;
+  height: 30px;
+  display: flex;
+}
+
+.see-project {
+  color: #396df2;
+  border: 1px solid #396df2;
+  border-radius: 5px;
+  padding: 3px;
+  background-color: #fff;
+  font-size: 20px;
+  margin-left: 55px;
+}
+
+.about-myself {
+  width: 100%;
+  height: 1100px;
+  background-color: #ffff;
+  margin-top: 80px;
+  border-top-right-radius: 100px;
+}
```

Comment:
- [ ] As mentioned in the project requirements Please use [`display: flex`](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)for the layout of your about myself section _**Make sure the content of the "About myself" section is aligned with flexbox.**_